scripts/compileBuiltins.js

```diff
@@ -17,7 +17,9 @@ glob(MARKDOWN_SRC, function(err, files) {
 
       return rawDescriptors.flatMap(string => {
         const name = string.split('\n')[0];
-        const description = marked.parse(string.split('\n').slice(1).join('\n'));
+        let description = marked.parse(string.split('\n').slice(1).join('\n'));
+
+        description = description.replace('<MarkerCamundaExtension></MarkerCamundaExtension>', '<b>Camunda Extension</b>');
```

Comment:
I'd rather expect this to be a heading `<h[n]>`.

Reply:
The "Camunda Extension" marker is can be placed within a paragraph (`p`) in the docs, making headings semantically incorrect

Reply:
So this one is a badge, correct? Not a heading then.

Reply:
Yes, it looks like this in the docs:

![image](https://github.com/bpmn-io/feel-editor/assets/21984219/4d1eb355-9eb9-42c9-8008-9890333868bb)

Reply:
Perhaps `strong` or `em` conveys the message of "this is important". The latter is even boldface per default.

Reply:
`em` seems to be semantically what we want, so let's use that one